clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -94,7 +94,7 @@ private Utils() {}
 
     // This matches URIs of formats: host:port and protocol:\\host:port
```

Comment:
nit: Is the comment correct?
`protocol:\\host:port` -> ` protocol://host:port` ?

Reply:
Thank you, this is a typo, I will fix it. 😄